tests/test_accessibility.py

```diff
@@ -31,25 +33,55 @@ def test_accessibility(test_obj):
             test_obj.accessibility_inject_axe()
             #Check if Axe is run in every page
             run_result = test_obj.accessibility_run_axe()
-            #Serialize dict to JSON-formatted string
-            result_str = json.dumps(run_result, ensure_ascii=False, separators=(',', ':'))
-            #Formatting result by removing \n,\\,timestamp
-            #Every test run have a different timestamp.
-            cleaned_result = re.sub(r'\\|\n|\r|"timestamp":\s*"[^"]*"', '', result_str)
-            #Compare Snapshot for each page
-            snapshot_result = test_obj.snapshot_assert_match(f"{cleaned_result}", f'snapshot_output_{page}.txt')
-            test_obj.conditional_write(snapshot_result,
-                                positive=f'Accessibility checks for {page} passed',
-                                negative=f'Accessibility checks for {page} failed',
-                                level='debug')
+            #Extract only the violations section
+            violations = run_result.get('violations', [])
+            #Snapshot file path to load
+            snapshot_file = os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
+                'snapshots',
+                'test_accessibility',
+                'test_accessibility',
+                'chrome',
+                f'snapshot_output_{page}.json'
+            )
+            saved_snapshot = snapshot_util.load_snapshot(snapshot_file)
+
+            # If snapshot does not exist, create a new one
+            if saved_snapshot is None:
```

Comment:
Probably we need to discuss this. Do we need to save snapshot if there is no snapshot?